src/runtime/request-helpers.ts

```diff
@@ -18,3 +18,14 @@ export const toFormData = (o: Record<string, any>): FormData => {
   Object.entries(o).forEach(([key, data]) => fd.append(key, data))
   return fd
 }
+
+export const toHeaders = (o: { [key: string]: any }): Record<string, any> => {
```

Comment:
Should be able to use `Record<string,string>` for typing I think?

Reply:
Actually I'm not sure this is even needed. We only filter undefined options for query params so we don't end up with `?blah=undefined`. It should be fine to just specify headers in the factory as `headers: { header1, header2 }`

Reply:
Because we have `Record<string, string>` on the Request interfaces, we can't just return the header values as they are, we need to convert them as strings. The other option is to have any header parameters as string variables in the client to convert them to string before calling any generated methods. This is a little less type safe though because it means people can pass any string in as a value rather than the type that's specified in the swagger doc.

Reply:
Just resolved our usage with @pleb